pkg/policies/opa/rego/docker/docker_run/yumupdate.rego

```diff
@@ -0,0 +1,8 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[run.id]{
+    run := input.run[_]
+	config := run.config
+    contains(config, ["yum update", "yum update-to", "yum upgrade", "yum upgrade-to"][_])
```

Comment:
value of `config` variable may or may not be in form of an array, need to create checks for both possibilities. please keep edge cases in mind while writing policy code